qotd/blog.md

```diff
@@ -0,0 +1,249 @@
+# Motivational QOTD with Courier and GPT2
+
+## Background
```

Comment:
I'd recommend adding either screenshots/gifs or embedding your demo video at the top.

Reply:
I've added the screenshot of the email. Let me know if more screenshots are needed.